### DISCUSSION TITLE
Settings seem to be lost on refresh

### DISCUSSION THREAD

**Discussion by Qwerty-Space** [vosmiic/jellyfin-ani-sync#123] [Q&A]
Hi,

I'm trying to get this plugin working, but every time I go to settings it seems to have forgotten almost everything.
I'm running on archlinux with jellyfin 10.8.13 and ani-sync version 3

---

Created issue #130

**Comment by vosmiic** [comment]
Sorry for not responding here, think I missed the email. This is a bug that needs fixing, I'll create an issue for this.
